common/lib/xmodule/xmodule/conditional_module.py

```diff
@@ -147,18 +183,27 @@ def get_html(self):
             'depends': ';'.join(self.required_html_ids)
         })
 
+    def author_view(self, context):
+        """
+        Renders the Studio preview by rendering each child so that they can all be seen and edited.
+        """
+        fragment = Fragment()
+        root_xblock = context.get('root_xblock')
+        is_root = root_xblock and root_xblock.location == self.location
+        if is_root:
```

Comment:
I see that this is a common convention, so the block just shows its message. Maybe that warrants a comment to make this clear.